### ISSUE TITLE
Exception thrown trying to pass PlayerJoinEvent to ChatBridge

### ISSUE THREAD

**Issue by ghost** [CraftTogether/Rinku#1]
```
[12:55:14 ERROR]: Could not pass event PlayerJoinEvent to ChatBridge v1.2-SNAPSHOT-2
org.json.JSONException: JSONArray[0] is not a JSONObject.
	at org.json.JSONArray.wrongValueFormatException(JSONArray.java:1682) ~[Rinku-1.0-SNAPSHOT-5-all.jar:?]
	at org.json.JSONArray.getJSONObject(JSONArray.java:481) ~[Rinku-1.0-SNAPSHOT-5-all.jar:?]
	at com.github.crafttogether.rinku.Rinku.retrieveAll(Rinku.java:30) ~[Rinku-1.0-SNAPSHOT-5-all.jar:?]
	at com.github.crafttogether.rinku.Rinku.find(Rinku.java:20) ~[Rinku-1.0-SNAPSHOT-5-all.jar:?]
	at com.github.crafttogether.chatbridge.minecraft.MinecraftJoinEvent.assignColor(MinecraftJoinEvent.java:41) ~[chat-bridge-1.2-SNAPSHOT-2-all.jar:?]
	at com.github.crafttogether.chatbridge.minecraft.MinecraftJoinEvent.onPlayerJoin(MinecraftJoinEvent.java:26) ~[chat-bridge-1.2-SNAPSHOT-2-all.jar:?]
	at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor2.execute(Unknown Source) ~[?:?]
	at org.bukkit.plugin.EventExecutor.lambda$create$1(EventExecutor.java:69) ~[paper-api-1.18.1-R0.1-SNAPSHOT.jar:?]
	at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:80) ~[paper-api-1.18.1-R0.1-SNAPSHOT.jar:git-Paper-147]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[paper-api-1.18.1-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:628) ~[paper-api-1.18.1-R0.1-SNAPSHOT.jar:?]
	at net.minecraft.server.players.PlayerList.postChunkLoadJoin(PlayerList.java:359) ~[paper-1.18.1.jar:git-Paper-147]
	at net.minecraft.server.players.PlayerList.lambda$placeNewPlayer$1(PlayerList.java:301) ~[paper-1.18.1.jar:git-Paper-147]
	at net.minecraft.server.network.ServerGamePacketListenerImpl.tick(ServerGamePacketListenerImpl.java:308) ~[?:?]
	at net.minecraft.network.Connection.tick(Connection.java:526) ~[?:?]
	at net.minecraft.server.network.ServerConnectionListener.tick(ServerConnectionListener.java:201) ~[?:?]
	at net.minecraft.server.MinecraftServer.tickChildren(MinecraftServer.java:1636) ~[paper-1.18.1.jar:git-Paper-147]
	at net.minecraft.server.dedicated.DedicatedServer.tickChildren(DedicatedServer.java:480) ~[paper-1.18.1.jar:git-Paper-147]
	at net.minecraft.server.MinecraftServer.tickServer(MinecraftServer.java:1470) ~[paper-1.18.1.jar:git-Paper-147]
	at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1264) ~[paper-1.18.1.jar:git-Paper-147]
	at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:317) ~[paper-1.18.1.jar:git-Paper-147]
	at java.lang.Thread.run(Thread.java:833) ~[?:?]
```

**Comment by ghost** [comment]
@M5rian The server is without chat bridging until you fix this

**Comment by m5rian** [comment]
Wow thanks for telling me after 12 hours ;)

**Comment by m5rian** [comment]
Fixed with b5f53696948c17f041e8b30afcd500b5f83dd020
